src/coreComponents/schema/docs/ExtendedDruckerPrager.rst

```diff
@@ -1,19 +1,20 @@
-defaultBulkModulus           real64 -1       Default Bulk Modulus Parameter                                       
-defaultCohesion              real64 0        Initial cohesion                                                     
-defaultDensity               real64 required Default Material Density                                             
-defaultDilationRatio         real64 1        Dilation ratio [0,1] (ratio = tan dilationAngle / tan frictionAngle) 
-defaultHardening             real64 0        Hardening parameter (hardening rate is faster for smaller values)    
-defaultInitialFrictionAngle  real64 30       Initial friction angle (degrees)                                     
-defaultPoissonRatio          real64 -1       Default Poisson's Ratio                                              
-defaultResidualFrictionAngle real64 30       Residual friction angle (degrees)                                    
-defaultShearModulus          real64 -1       Default Shear Modulus Parameter                                      
-defaultYoungModulus          real64 -1       Default Young's Modulus                                              
-name                         string required A name is required for any non-unique nodes                          
-============================ ====== ======== ==================================================================== 
+================================== ====== ======== ==================================================================== 
+Name                               Type   Default  Description                                                          
+================================== ====== ======== ==================================================================== 
+defaultBulkModulus                 real64 -1       Default Bulk Modulus Parameter                                       
+defaultCohesion                    real64 0        Initial cohesion                                                     
+defaultDensity                     real64 required Default Material Density                                             
+defaultDilationRatio               real64 1        Dilation ratio [0,1] (ratio = tan dilationAngle / tan frictionAngle) 
+defaultHardening                   real64 0        Hardening parameter (hardening rate is faster for smaller values)    
+defaultInitialFrictionAngle        real64 30       Initial friction angle (degrees)                                     
+defaultPoissonRatio                real64 -1       Default Poisson's Ratio                                              
+defaultResidualFrictionAngle       real64 30       Residual friction angle (degrees)                                    
+defaultShearModulus                real64 -1       Default Shear Modulus Parameter                                      
+defaultThermalExpansionCoefficient real64 0        Default Thermal Expansion Coefficient                                
```

Comment:
we should clarify that this is the linear thermal expansion coefficient, not the volumetric thermal expansion coefficient. They are differed by a factor 3.